dbsync.py

```diff
@@ -801,3 +802,32 @@ def dbsync_push(mc):
 def dbsync_status(mc):
     for conn in config.connections:
         status(conn, mc)
+
+
+def clean(conn_cfg, mc):
+    from_db = config.init_from.lower() == "db"
+
+    if pathlib.Path(config.working_dir).exists():
+        try:
+            shutil.rmtree(config.working_dir)
+        except FileNotFoundError as e:
+            raise DbSyncError("Unable to remove working directory: " + str(e))
+
+    try:
+        conn_db = psycopg2.connect(conn_cfg.conn_info)
+    except psycopg2.Error as e:
+        raise DbSyncError("Unable to connect to the database: " + str(e))
+
+    try:
+        _drop_schema(conn_db, conn_cfg.base)
+
+        if not from_db:
+            _drop_schema(conn_db, conn_cfg.modified)
+
+    except psycopg2.Error as e:
+        raise DbSyncError("Unable to drop schema from database: " + str(e))
+
```

Comment:
I believe if we have init from DB, we should also remove geopackage in the MM project

Reply:
Did some tests on that. We should definitely clear it, but the question is how. If we just delete the file, then DB Sync Init will not work as it is missing the sync file. So this a no go.

I can see two potential solutions to the issue.
1. We can just create a empty file. That kind of works, but if the use tries to open in QGIS while the file empty, QGIS complains that the file is not valid (it is missing some tables). I would avoid that.
2. We can use Python's `sqlite3` to connect to GPKG and clear tables and data in such way, that we leave empty yet valid GPKG.
3. We could have empty valid GPKG as part of DB Sync and when the clean is done, we would just override the old file with this prepared one.

I am quite fond of the solution number 3, but mainly because it is simple to implement ;-)